test/e2e/api_inheritance/api_inheritance_test.go

```diff
@@ -18,49 +18,55 @@ package api_inheritance
 
 import (
 	"context"
+	"embed"
```

Comment:
(No action required) I'm also rewriting this test (for shared fixture #636) so I guess there's a messy rebase in my future.

Reply:
This one was necessary. The test was just wrong.